### DISCUSSION TITLE
iOS 18. Is this normal behavior?

### DISCUSSION THREAD

**Discussion by 0zd0** [KingSora/OverlayScrollbars#686] [Q&A]
https://github.com/user-attachments/assets/d9a7eee7-3fce-4f1c-8cfd-c68fbb4cfdc6

**Comment by KingSora** [comment]
@0zd0 good day :)

Generally speaking, any behavior that is not observed with native scrollbars is considered "not normal".

Is the same behavior observed on android / other browsers?

I could take a look at your case if you can point me to an working example (or the real app)
It looks like you have 2 nested overflowing containers where one eats up the other one therefore clipping its content
